application/docker-compose.yml

```diff
@@ -9,12 +9,19 @@ services:
       - POSTGRES_USER=stratospheric
       - POSTGRES_PASSWORD=stratospheric
       - POSTGRES_DB=stratospheric
+  activemq:
+    image: bjoernkw/activemq-docker-image
```

Comment:
Using a Docker Image from us brings maintenance effort on our side. Is the _official_ ActiveMQ Docker image not sufficient enough or lacking important configuration?

Reply:
There's only an official image for ActiveMQ Artemis https://hub.docker.com/r/apache/activemq-artemis . However, Amazon MQ uses ActiveMQ 5 "Classic", which behaves differently in some respects. There are plenty of ActiveMQ 5 "Classic" Docker images out there. Most of those aren't maintained regularly, though. For this reason and in order to be able to use the specific ActiveMQ version used by Amazon MQ for the local instance, too, I thought it best to create our own image.

Reply:
Okay, that's a valid reason :+1: